cri-o-fedora/Dockerfile

```diff
@@ -13,7 +13,7 @@ LABEL com.redhat.component="cri-o" \
 
 COPY README.md /
 
-RUN dnf install --setopt=tsflags=nodocs -y iptables cri-o iproute runc && \
+RUN dnf install --enablerepo=updates-testing --setopt=tsflags=nodocs -y iptables cri-o iproute runc && \
```

Comment:
Why enable testing?

Reply:
Oh I see, it's temporary to get the latest stable ``cri-o``. Could this be done on it's own line to avoid getting testing versions of the other packages?

Reply:
some of the packages are updated in Fedora, some others are not.  I always end up with build issues when a new release is out.  This was the suggestion I got from: @lsm5

Reply:
OK, let's just make sure we revert back to stable once the package has moved.